### PR TITLE
Added optional MODSEC_DISABLE_BACKEND_COMPRESSION env variable

### PR DESCRIPTION
**Pull request by karelorigin**
The ModSecurity `SecDisableBackendCompression` could not be configured in this docker image. This PR adds that option.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -189,6 +189,7 @@ All these variables impact in configuration directives in the modsecurity engine
 | MODSEC_DATA_DIR  | A string indicating the path where persistent data (e.g., IP address data, session data, and so on) is to be stored (Default: `/tmp/modsecurity/data`) | 
 | MODSEC_DEBUG_LOG  | A string indicating the path to the ModSecurity debug log file (Default: `/dev/null`) | 
 | MODSEC_DEBUG_LOGLEVEL  | An integer indicating the verboseness of the debug log data (Default: `0`). Accepted values: `0` - `9`. See [SecDebugLogLevel](https://github.com/SpiderLabs/ModSecurity/wiki/Reference-Manual-(v2.x)#secdebugloglevel). | 
+| MODSEC_DISABLE_BACKEND_COMPRESSION  | A string indicating whether or not to disable backend compression (Default: `Off`). Allowed values: `On`, `Off`. See [SecDisableBackendCompression](https://github.com/SpiderLabs/ModSecurity/wiki/Reference-Manual-(v2.x)#secdisablebackendcompression) for more |
```
Comment (theseion) [inline README.md:192]:
```suggestion
| MODSEC_DISABLE_BACKEND_COMPRESSION  | A string indicating whether or not to disable backend compression (Default: `Off`). Allowed values: `On`, `Off`. See [SecDisableBackendCompression](https://github.com/SpiderLabs/ModSecurity/wiki/Reference-Manual-(v2.x)#secdisablebackendcompression) for more. Only supported in ModSecurity 2.x, will have not effect on 3.x |
```

**Review by theseion — CHANGES_REQUESTED** [review]
Thanks! Looks good, except that I'd like the comment to point out that it's for 2.x only.

File: README.md (hunk 1 above)
Comment (theseion) [reply README.md:192]: Yes :D

**Comment by fzipi** [comment]
Maybe we can add some shell scripting to only generate the variable if running on apache. This error doesn't look nice, and might confuse users:
```
nginx-nightly            | nginx: [emerg] "modsecurity_rules_file" directive Rules error. File: /etc/modsecurity.d/modsecurity.conf. Line: 224. Column: 19. SecDisableBackendCompression is not supported.  in /etc/nginx/conf.d/modsecurity.conf:2
```

If we can live with that, then no problems

**Comment by theseion** [comment]
Good point. I agree that we should remove it from httpd.

File: README.md (hunk 1 above)
Comment (fzipi) [reply README.md:195]: @theseion Should we correct to "no" then? This is the last change for this to be merged.

Comment (theseion) [reply README.md]: Done

**Comment by theseion** [comment]
I've tried to recreate the issue you had with nginx and the variable but I can't. I believe it was already solved with additional `RUN` instruction that comments the directive for v3.

I've rebased onto develop, so that builds should run again.

**Review by theseion — APPROVED** [review]

**Review by fzipi — APPROVED** [review]
LGTM also

**Review by fzipi — APPROVED** [review]
LGTM.

**Pull request merged**